### DISCUSSION TITLE
Include a option for reverting back to older updates?

### DISCUSSION THREAD

**Discussion by Muhammad-Mohid-Khan** [zen-browser/desktop#3550] [Ideas]
There should be an option for reverting back to older updates this new update has a lot of bugs and maybe if theres a way to go back to the old update it would be nice for now and then when the release is stable we can update back later 
i mindlessly updated it not releasing there would be so many bugs ( indicated by the issues [643] is insane )

**Comment by mateuszkozakiewicz** [comment]
Can you not go to https://github.com/zen-browser/desktop/releases and download the old version?
